pyproject.toml

```diff
@@ -30,6 +30,7 @@ toml = ">=0.10.2"
 pytest = "^7.1.1"
 pytest-mock = "^3.7.0"
 cfn-lint = "^0.75.1"
+exasol-toolbox = "^0.12.0"
```

Comment:
for what are you using the toolbox?

Reply:
I wanted to use it for automatic release PR creation (which adapts the changelog and versions). But I didn't manage to use it....
So, I removed the dependency again.